src/App.js

```diff
@@ -54,6 +56,184 @@ const App = () => {
           </div>
         </div>
       </section>
+      <section className="digits">
+        <h2 className="digits__title">Цифры и факты</h2>
+        <h3 className="digits__subtitle">Про учебу и мозг</h3>
+        <table className="digits__table">
+          <tr>
+            <td>
+              <h4 className="table__title">86 миллиардов</h4>
```

Comment:
change classes in this section as it goes in the rest. For example "digits__table-title"

Reply:
done

Reply:
Have you pushed it? I still see `table__title` and the rest

Reply:
oh, my bad, I haven't cvhanged it everywhere. Fixed, will be in the next version.